Assets/MRTK/SDK/Editor/Inspectors/Utilities/MeshOutline/BaseMeshOutlineInspector.cs

```diff
@@ -0,0 +1,157 @@
+
+                if (GUILayout.Button("Update material settings to default"))
+                {
+                    ForceUpdateToDefaultOutlineMaterial(ref currentMat);
+                }
+            }
+
+            // Draw other properties
+            DrawPropertiesExcluding(serializedObject, nameof(m_Script), nameof(outlineMaterial));
+
+            if (EditorGUI.EndChangeCheck())
+                serializedObject.ApplyModifiedProperties();
+        }
+
+        private Material CreateNewMaterial()
+        {
+            var material = new Material(Shader.Find(MrtkStandardShaderName));
+            ForceUpdateToDefaultOutlineMaterial(ref material);
+            AssetDatabase.CreateAsset(material, $"Assets/{Selection.activeGameObject.name}Mat.mat");
+            return material;
+        }
+
+        private void ForceUpdateToDefaultOutlineMaterial(ref Material material)
+        {
+            if (!IsCorrectShader(material))
+            {
+                material.shader = Shader.Find(MrtkStandardShaderName);
+            }
+
+            foreach (var pair in defaultOutlineMaterialSettings)
+            {
+                switch (pair.Value.GetType().Name)
+                {
+                    case nameof(System.Single):
+                        material.SetFloat(pair.Key, (float)pair.Value);
+                        break;
+                    case nameof(System.Boolean):
+                        var val = (bool)pair.Value;
+                        if (val) 
+                        {
+                            material.EnableKeyword(pair.Key);
+                        }
+                        else
+                        {
+                            material.DisableKeyword(pair.Key);
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            }
+        }
+
+        private bool IsCorrectMaterial(Material material)
+        {
+            if (!IsCorrectShader(material))
+            {
+                return false;
+            }
+
+            return defaultOutlineMaterialSettings.All(x => 
+            {
+                switch (x.Value.GetType().Name)
+                {
+                    case nameof(System.Single):
+                        return material.GetFloat(x.Key) == (float)x.Value;
+                    case nameof(System.Boolean):
+                        var val = (bool)x.Value;
+                        if (val)
+                        {
+                            return material.IsKeywordEnabled(x.Key);
+                        }
+                        else
+                        {
+                            return !material.IsKeywordEnabled(x.Key);
+                        }
+                    default:
+                        // Default return value
+                        return false;
+                }
+            });
+        }
+
+        private bool IsCorrectShader(Material material)
```

Comment:
nit: You could replace this with [IsUsingMrtkStandardShader()](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/70d5731d959388a199143008d1c8ce7c4c4c577d/Assets/MRTK/Core/Utilities/StandardShader/StandardShaderUtility.cs#L46).

Reply:
Thank you!